users/chart/templates/deployment.yaml

```diff
@@ -93,6 +93,16 @@ spec:
             secretKeyRef:
               name: users
               key: ADMIN_CLIENT_SECRET_DEV1
+#        - name: KEYCLOAK_ROLE_ClientViewsAllUsers_ID
```

Comment:
Why these fields was commented?

Reply:
It is commented up to the point our DevOps engineer will put those values in 'secret' on dev environments.
It will be uncommented and used after that